app.test.js

```diff
@@ -46,4 +46,41 @@ describe('getCategories', () => {
             })
           });
       });
+})
+
+describe('getReviewsById', () => {
+  test("Responds with a review object with correct properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.review).toMatchObject({
+          review_id: expect.any(Number),
```

Comment:
Also make sure that the array length is  > 0 in this test, as empty arrays can sneak by otherwise.

Reply:
But the test only asks for an object not an array